.github/ISSUE_TEMPLATE/documentation_issue.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Documentation issue
+about: Report issues with the Microsoft Edge documentation
+title: ''
+labels: ''
+assignees: ''
+---
+
+<!--
+⚠ If your feedback is not about the Microsoft Edge developer documentation, please consider submitting it here instead:
+
+* For DevTools: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/contact
+* For WebView2: https://github.com/MicrosoftEdge/WebView2Feedback/issues
+-->
+
+**Link to the documentation page**: the URL to the page.
```

Comment:
I'd capitalize "The", "A", "If", and "If", like in a table.  I wonder if a style guide addresses this.  This is documentation of UI controls.  Imagine a table format.  I'd consider " - " instead of ": " as separator; not sure if doc'd in style guides.